fare/file_management/subjects.json

```diff
@@ -0,0 +1,19 @@
+{
+  "subject":[
```

Comment:
Watch out: this should not be a flat list. Checking the old fare, each of those subject contain other sub-subjects: 
```
t('Scienze') => drupal_map_assoc(array(
      t('Scienze naturali'),
      t('Scienze astronomiche'),
    )),
```
Let's map them here in JSON. 
Of course this means that also our database should not be flat but it must reflect this structure.